src/p2p/Join/index.ts

```diff
@@ -566,7 +604,7 @@ export function parseRecord(record: P2P.CycleCreatorTypes.CycleRecord): P2P.Cycl
         if (record.finishedSyncing.includes(nodeId)) continue
 
         // add node to lostAfterSelection to be added to the cycle record next cycle
-        lostAfterSelection.push(nodeId)
+        lostAfterSelection.push({ nodeId })
```

Comment:
im curious about the point of this change

Reply:
looking at the lib-types changes more closely, i get it now. once we go back to the old way of handling lostAfterSelection, ig this will go back to the way it was